styles.css

```diff
@@ -1 +1,78 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+* {
+    padding: 0px;
+    margin: 0px;
+    background-color: #FFEFDB;
+
+
+}
+.grid {
+    display: grid;
+    color: #ffffff;
+    grid-template-columns: repeat(3,1fr);
+    grid-template-rows: repeat(12, 50px);
+    margin: 16px;
+    grid-gap: 16px;
+}
+
+#a {
+    display: grid;
+    background-color: #BA3B44;
+    justify-content: center;
+    align-content: center;
+    grid-column: 1/2;
+    grid-row: 1/2
+}
+#b {
+    display: grid;
+    background-color: #BA3B44;
+    justify-content: center;
+    align-content: center;
+    grid-column: 1/2;
+    grid-row: 2/11;
+    position: relative;
+}
+#b p {
+    display: grid;
+    background-color: #BA3B44;
+    position: absolute;
+
+}
+#c {
+    display: grid;
+    background-color: #BA3B44;
+    justify-content: center;
+    align-content: center;
+    grid-column: 2/4;
+    grid-row:1/6;
+
+
+}
+#d {
+    display: grid;
+    background-color: #BA3B44;
+    justify-content: center;
+    align-content: center;
+    grid-column: 3/4;
+    grid-row: 6/8;
+    grid-template-columns: minmax(50px,auto);
+
+}
+#e {
+    display: grid;
+    background-color: #BA3B44;
+    justify-content: center;
+    align-content: center;
+    grid-column: 2/3;
+    grid-row: 6/11;
+
+}
+#f {
+    display: grid;
+    background-color: #BA3B44;
+    justify-content: center;
+    align-content: center;
+    grid-column: 1/4;
+    grid-row: 11/12
+}
+
```

Comment:
Ikzelf kijk veel tutorials op youtube, vooral die van Traversy Media zijn erg leerzaam.
Daar staat ook een hele fijne over CSS gid: https://www.youtube.com/watch?v=jV8B24rSN5o

Misschien dat je daar misschien wat meer duidelijkheid over krijg. Het is vooral een kwestie van oefenen,oefenen,oefenen.

Reply:
Dag Thieu, 
Dank voor je comments en tips! Ik zat zeker vast met die grid area's . Ik bleef maar foutmeldingen kregen daarover. Er gaat iets verkeerd maar ik ben er nog niet achter wat. Ik ga je youtube link zeker bekijken . Dank voor je tijd.
grt Dhiraj